extensions/wikidata/module/langs/translation-en.json

```diff
@@ -126,6 +126,10 @@
     "warnings-messages/no-edit-generated/body": "There might be something wrong with your schema.",
     "warnings-messages/no-issue-detected/title": "No issue was detected in your edits.",
     "warnings-messages/no-issue-detected/body": "Note that OpenRefine cannot detect all the types of problems Wikidata edits can have.",
+    "warnings-messages/multi-valued-property-is-required-for-new-item/title": "{property_entity} should have more than one statements.",
```

Comment:
"one statements" -> "one statement". Also, it could be good to change the title to something like `{property_entity} should have more than one statement on new items` to make it clear what the difference is with the second issue. (This is mostly useful if both issues get reported at the same time).

The same applies to the other issue title.